lib/persistence/sqlite_adaptor.rb

```diff
@@ -7,8 +7,9 @@ def self.database
 
   def self.create_datablase
     self.database.create_table(:following) do
-      Fixnum :user_id
-      Time   :created_at
+      Integer   :user_id
+      Time      :created_at
+      TrueClass :unfollowed
```

Comment:
It's weird that Sequel doesn't have `NilClass`.
http://sequel.jeremyevans.net/rdoc/files/doc/schema_modification_rdoc.html#label-Column+types

In this case, I think we can specify `FalseClass` because false is going to be the default value. I mean it doesn't matter because this type is used to specify that is a boolean column but it will make more sense when I will take a look in the future and I won't remember the details.